static/src/javascripts/bootstraps/enhanced/liveblog.js

```diff
@@ -95,8 +97,14 @@ const initTracking = () => {
 
 const trackOphanClick = (pinnedBlockId, clickValue) => {
 	ophan.record({
-		...componentEvent(pinnedBlockId, 'CLICK'),
-		value: clickValue,
+        componentEvent: {
```

Comment:
I think we can still use the componentEvent helper function here no?

Reply:
The value needs to be nested within the componentEvent object so for I've inlined here to be clearer.

Reply:
Ahh I see, could we make that an optional param?

Reply:
Yep, that's nicer. Updated :)